### PR TITLE
Ticket 7

### PR DESCRIPTION
**Pull request by gracefaz**
Ticket 7: Added Vote (and Deduct Vote) buttons which increase and decrease the number of votes respectively.

### CODE REVIEW

File: src/components/SingleReview.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ export const SingleReview = () => {
     patchReviewById(review_id);
   };
 
+  const handleDeductVoteClick = () => {
+    setCount((currCount) => currCount - 1);
```
Comment (popatre) [inline src/components/SingleReview.js:23]: Deduct vote decrements client side, but actually increments in the database. 
If you refresh after deducting some, its actually increased the database votes

**Review by popatre — CHANGES_REQUESTED** [review]

File: src/components/SingleReview.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { fetchReviewById } from "../utils.js/apiCalls";
+import { fetchReviewById, patchReviewById } from "../utils.js/apiCalls";
 
 export const SingleReview = () => {
   const [review, setReview] = useState({});
+  const [count, setCount] = useState(0);
+  // const [click, setClick] = useState(false);
```
Comment (Rosem10) [inline src/components/SingleReview.js]: Remove if not using, please!

File: src/utils.js/apiCalls.js [hunk 3]
```diff
@@ -30,3 +30,11 @@ export const fetchReviewById = (review_id) => {
     return response.data.review;
   });
 };
+
+export const patchReviewById = (review_id, countInc) => {
+  const body = { inc_votes: countInc };
+  return ncGamesAPI.patch(`/reviews/${review_id}`, body).then((response) => {
+    console.log(response.data);
```
Comment (Rosem10) [inline src/utils.js/apiCalls.js]: Remove this too please.

**Review by Rosem10 — APPROVED** [review]
I really like your styling!

**Pull request merged**